[박영서] Mission 2: 모델링

yeongseoPark opened the pull request:
>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
(여기에 수정 사항에 대한 간략한 한줄 요약/제목 작성해 주세요.)
users.models.py 구현
		 
## 수정 사항들 자세한 내용
(2/17 17:50)
git diff를 통해 git 으로 제출되는 코드의 정렬을 맞췄습니다
class Users와 class Meta 사이의 간격을 한 줄 두었습니다


## 기타 질문 및 특이 사항
(궁금한 사항들, 하면서 느낀 점들 공유해주실 것이 있으면 자유롭게 작성해 주세요.)

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
조건을 만족한다면 각 빈칸에 x 표시 해주세요
- [ X] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [ X] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [ X] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [ X] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

yeongseoPark committed:
Add : User model

kyeongraekim commented on students/yeongseo/westagram/users/models.py:
```diff
@@ -0,0 +1,10 @@
+
+# Create your models here.
+class Users(models.Model):
+    name = models.CharField(max_length=20)
```
id(흔히 한국인들이 사용하는 계정의 명칭) → username
name → first_name, last_name

위와 같이 기획해 보는 것은 어떨까요?

yeongseoPark committed:
Modify : User model

kyeongraekim commented on students/yeongseo/westagram/users/models.py:
```diff
@@ -0,0 +1,10 @@
+# Create your models here.
+class Users(models.Model):
+    name = models.CharField(max_length=20)
+    email = models.EmailField(max_length=40)
```
email field에 unique 속성을 추가하셔서 유저가 중복 저장되는 문제가 발생하지 않도록 수정해주세요!
공식 문서에서 그 방법을 찾아보고 적용해주세요.

kyeongraekim commented on students/yeongseo/westagram/users/models.py:
```diff
@@ -0,0 +1,10 @@
+class Users(models.Model):
+    name = models.CharField(max_length=20)
+    email = models.EmailField(max_length=40)
+    password = models.CharField(max_length=40)
```
password property는 나중에 해시(암호화)된 비밀번호 길이를 고려해서 넉넉하게 100 이상으로 설정해주세요.

kyeongraekim commented on students/yeongseo/westagram/users/models.py:
```diff
@@ -0,0 +1,10 @@
+    name = models.CharField(max_length=20)
+    email = models.EmailField(max_length=40)
+    password = models.CharField(max_length=40)
+    phone_number = models.CharField(max_length=16) #+82등 국가번호 붙는 경우 감안하여 char로 필드 지정
```
코드상으로 충분히 파악이 가능한 불필요한 주석은 삭제해주세요!~

kyeongraekim reviewed (changes requested):
아래내용 확인하여 수정해주세요!~

kyeongraekim commented on students/yeongseo/westagram/users/models.py:
```diff
@@ -0,0 +1,11 @@
+    email             = models.EmailField(max_length=40)
+    password        = models.CharField(max_length=40)
+    phone_number = models.CharField(max_length=16) 
+    #+82등 국가번호 붙는 경우 감안하여 char로 필드 지정
```
사용자/회원 데이터의 이력 관리를 위해서 created_at, updated_at column을 추가해주세요.

created_at = models.DateTimeField(auto_now_add=True)
updated_at = models.DateTimeField(auto_now=True)

yeongseoPark committed:
name -> username
added unique=True to email
password max_length modified
added created_at , updated_at column

kyeongraekim commented on students/yeongseo/westagram/users/models.py:
```diff
@@ -0,0 +1,12 @@
+
+# Create your models here.
+class Users(models.Model):
+    username        = models.CharField(max_length=30)
```
깃헙상에 현재 지그재그로 변수명 컨벤션이 맞추어져있네요. 일자 재정렬 부탁드립니다!~
git add . 전에 git diff 를 찍어보시면 실제 깃헙상에 올라갈 코드들의 모습들이 보일텐데 참고해주세요!

kyeongraekim commented on students/yeongseo/westagram/users/models.py:
```diff
@@ -0,0 +1,12 @@
+    phone_number = models.CharField(max_length=16) 
+    created_at       = models.DateTimeField(auto_now_add=True)
+    updated_at      = models.DateTimeField(auto_now=True)
+    class Meta:
```
class Meta 선언에 앞서서 한 줄 라인 공백을 띄어주세요

kyeongraekim reviewed (changes requested):
아래 내용 참고하여 수정해주세요!~

yeongseoPark committed:
aligned code
space between class Users / class Meta

kyeongraekim reviewed (approved):
LGTM!~
